ci/travis/install-dependencies.sh

```diff
@@ -346,7 +346,9 @@ install_dependencies() {
 
   # Additional RLlib test dependencies.
   if [ "${RLLIB_TESTING-}" = 1 ] || [ "${DOC_TESTING-}" = 1 ]; then
-    pip install -r "${WORKSPACE_DIR}"/python/requirements/rllib/requirements_rllib.txt
+    pip install -r "${WORKSPACE_DIR}"/python/requirements/ml/requirements_rllib.txt
+    #TODO(amogkam): Add this back to requirements_rllib.txt once mlagents no longer pins torch version.
+    pip install mlagents==0.27
```

Comment:
sven seemed to suggest we need to --nodep here?

Reply:
If we do `--no-dep` it will use this flag for all dependencies listed in the requirements file. I thought it would be better to just move out this one dependency rather than change the install behavior for all the dependencies in the file.

If we move out `mlagents`, then `--no-dep` is no longer needed.

Reply:
oh definitely. what I meant was whether we need --no-dep here when we install this single mlagents lib.
but all good, if everything works.